Kuroba/app/src/main/res/values/strings.xml

```diff
@@ -505,20 +505,20 @@ Note that for country code filtering, troll country codes should be prepended wi
 
     <string name="setting_save_thread_folder_description">Create a folder for each thread when saving</string>
     <string name="setting_save_server_filename">Save server filename</string>
-    <string name="setting_save_server_filename_description">
-"Save the image with the filename the site assigned.
-If disabled, save the image with the filename the uploader assigned."
-    </string>
+    <string name="setting_save_server_filename_description">Save the image with the filename the site assigned. If disabled, save the image with the filename the uploader assigned.</string>
     <string name="setting_video_default_muted">Start videos muted</string>
-    <string name="setting_video_default_muted_description">If a video has audio, mute it by default.</string>
+    <string name="setting_video_default_muted_description">If a video has audio, mute it by default</string>
     <string name="setting_video_open_external">Play videos with external player</string>
     <string name="setting_video_open_external_description">Play videos in an external media player app</string>
     <string name="setting_share_url">Share URL to image</string>
     <string name="setting_share_url_description">Share the URL to the image instead of the image itself</string>
     <string name="settings_reveal_image_spoilers">Reveal image spoilers</string>
-    <string name="settings_reveal_image_spoilers_description">Always reveal spoiler thumbnails and images.</string>
+    <string name="settings_reveal_image_spoilers_description">Always reveal spoiler thumbnails and images</string>
     <string name="settings_allow_media_scanner_scan_local_threads_title">Allow the Android media scanner to scan images from locally saved threads</string>
-    <string name="settings_allow_media_scanner_scan_local_threads_description">If enabled, images that were downloaded along with threads will appear in other applications, like galleries, messengers etc; You may have to restart your phone or wait for the changes to kick in</string>
```

Comment:
strings.xml doesn't automatically put in newlines per the formatting of the file. You will have to specify a `\n` if you want it to display with a newline.

Reply:
Well, yeah, I just figured I'd make it so you don't have to constantly scroll all the way to the right to read lines. The point wasn't to put it on a newline necessarily.